cygnss_wetlands/utils/constants.py

```diff
@@ -1,5 +1,7 @@
 CYGNSS_WAVELENGTH_M = 0.19  # meters
 
 EARTH_RADIUS_KM = 6371  # average radius of Earth, km
+CYGNSS_DISTANCE_TRAVELLED_IT_KM = 6  # km travelled by spacecraft during integration time
```

Comment:
Where is this coming from?

Reply:
This is found in the literature (e.g. https://www.mdpi.com/2072-4292/12/8/1317#) -- standard CYGNSS data output rate is 1 Hz resulting in surface integration of ~6 km in the along track direction due to satellite motion

Reply:
It may be worthwhile to comment the link in there indicating this

Reply:
I've updated the comments to give more detail on these distances